zkevm-circuits/src/table.rs

```diff
@@ -153,14 +152,30 @@ impl TxTable {
                     .collect();
                 for tx in txs.iter().chain(padding_txs.iter()) {
                     for row in tx.table_assignments(randomness) {
-                        for (column, value) in tx_table_columns.iter().zip_eq(row) {
-                            region.assign_advice(
-                                || format!("tx table row {}", offset),
-                                *column,
-                                offset,
-                                || Value::known(value),
-                            )?;
-                        }
+                        region.assign_advice(
+                            || format!("tx table row {}", offset),
+                            self.tx_id,
+                            offset,
+                            || Value::known(row[0]),
+                        )?;
+                        region.assign_fixed(
+                            || format!("tx table row {}", offset),
+                            self.tag,
+                            offset,
+                            || Value::known(row[1]),
+                        )?;
+                        region.assign_advice(
+                            || format!("tx table row {}", offset),
+                            self.index,
+                            offset,
+                            || Value::known(row[2]),
+                        )?;
+                        region.assign_advice(
+                            || format!("tx table row {}", offset),
+                            self.value,
+                            offset,
+                            || Value::known(row[3]),
+                        )?;
```

Comment:
Perhaps still nicer to use a for loop here for all the advice columns.

Reply:
Updated. Use a for loop for all the advice columns.